.github/workflows/run-tests.yml

```diff
@@ -111,11 +111,12 @@ jobs:
           path: ~/downloads/tarball
       - name: Echo tarball download location
         run: echo ${{ steps.download.outputs.download-path }}
+        # TODO: update ASAP after April Release
       - name: Install Tarball
         shell: bash
         run: |
           # We need to determine the Tarball's name first.
-          TARBALL_NAME=$(ls ~/downloads/tarball | grep salesforce-plugin-code-analyzer-5\\.0\\.0-beta\\.[0-9]*\\.tgz)
+          TARBALL_NAME=$(ls ~/downloads/tarball | grep salesforce-plugin-code-analyzer-.*\\.tgz)
```

Comment:
I recommend `grep`ing for `salesforce-plugin-code-analyzer-5\\.[0-9]*\\.[0-9]*\\.tgz`.

Reply:
This one I realized we'll need to wait until after we go GA, but we can clean that up then if you'd like 👍